extensions/wikia/UserRenameTool/UserRenameTask.class.php

```diff
@@ -1,225 +0,0 @@
-<?php
```

Comment:
Renamed/split into tasks/WikiRenameBase.class.php, tasks/MultiWikiRename.class.php and tasks/SingleWikiRename.class.php